README.md

```diff
@@ -20,66 +22,24 @@ To vote on AIPs impacting the state of the Blockchain, go to https://governance.
-| [AIP 39](https://github.com/aptos-foundation/AIPs/issues/173)| Core | [Separate gas payer](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-39.md) | Standard | Draft |
-| [AIP 40](https://github.com/aptos-foundation/AIPs/issues/178)| Core | [Address Standard v1](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-40.md) | Standard | Draft |
-| [AIP 41](https://github.com/aptos-foundation/AIPs/issues/185)| Framework | [Move APIs for randomness generation](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-41.md) | Standard | Draft |
-| AIP 42 | Core | [Aptos TypeScript SDK V2](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-42.md) | Standard | Draft |
-| [AIP 43](https://github.com/aptos-foundation/AIPs/issues/209) | Framework | [Parallelize Digital Assets (Token V2) minting/burning](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-43.md) | Standard | Draft |
-| [AIP 44](https://github.com/aptos-foundation/AIPs/issues/200) | Core | [Module Events](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-44.md) | Standard | Draft |
-| [AIP 46](https://github.com/aptos-foundation/AIPs/issues/222) | Framework | [New modules for ElGamal, Pedersen and Bulletproofs over Ristretto255](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-46.md) | Standard | Draft |
-| [AIP 47](https://github.com/aptos-foundation/AIPs/issues/226) | Framework | [Aggregators V2 ](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-47.md) | Standard | Draft |
-| [AIP 48](https://github.com/aptos-foundation/AIPs/issues/237) | Framework | [Allow direct commission change vesting contract](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-48.md) | Standard | Draft |
-| [AIP 49](https://github.com/aptos-foundation/AIPs/issues/247) | Framework | [secp256k1 ECDSA for Transaction Authentication](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-49.md) | Standard | Draft |
-| [AIP 50](https://github.com/aptos-foundation/AIPs/issues/249) | Framework | [Change Commission rates in delegation pools](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-50.md) | Standard | Draft |
-| [AIP 51](https://github.com/aptos-foundation/AIPs/issues/251) | Framework | [Changing beneficiaries for operators ](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-51.md) | Standard | Draft |
-| [AIP 52](https://github.com/aptos-foundation/AIPs/issues/258) | Framework | [Automated Account Creation for Sponsored Transactions](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-52.md) | Standard | Draft |
-| [AIP 53](https://github.com/aptos-foundation/AIPs/issues/257) | Framework | [Make Fee Payer Address Optional in Transaction Authenticator](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-53.md) | Standard | Draft |
-| ... | ... ||||
+| Number | Theme | Title  | Author |
```

Comment:
Can you add a comment before the table explaining that this is a subset of (important?) AIPs from the AIP list on the wiki included for convenience? Assuming that's what this is.